data/steve_s_factory_manager/mod.cg

```diff
@@ -0,0 +1,10 @@
+schema: 1
+
+name:             Steve's Factory Manager
+author:           Vswe
+description:      No description specified by developer.
```

Comment:
Go ahead and take a crack at this yourself.  Very few mods have a nice short description, so I've written most of the ones on the site already.